src/server/views/api/jobRetry.js

```diff
@@ -10,7 +10,19 @@ async function handler(req, res) {
   if (!job) return res.status(404).send({error: 'job not found'});
 
   try {
-    await job.retry();
+    let jobState;
+    if (queue.IS_BEE) {
+      jobState = job.status;
+    } else {
+      jobState = await job.getState();
+    }
```

Comment:
truly just a suggestion - not a necessary change
```suggestion
    const jobState = queue.IS_BEE ? job.status : await job.getState();
```